packages/web-app-files/tests/unit/components/SideBar/Links/PublicLinks/LinkActions.spec.js

```diff
@@ -0,0 +1,147 @@
+import LinkActions from '../../../../../../src/components/SideBar/Links/PublicLinks/LinkActions.vue'
```

Comment:
You can try this one
```suggestion
import LinkActions from '@files/src/components/SideBar/Links/PublicLinks/LinkActions.vue'
```